backend/src/core/enums/gender.py

```diff
@@ -0,0 +1,6 @@
+from src.core.enums.base import BaseEnum
+
+
+class GenderTypeEnum(BaseEnum):
```

Comment:
M, W. Make sense decrease length of choices if it's possible to keep less information in a database

Reply:
Okay